.github/pull_request_template.md

```diff
@@ -0,0 +1,11 @@
+## Log
+
+Provide a list of changes and describe used solutions referencing issues
```

Comment:
well, imho, you can check changes in commits sections

Reply:
only if commit history is good and it's usually not

Reply:
and it will be the first review-comment, that commits history should be recreated

Reply:
Still there is nothing bad in keeping all changes on top of the page as they can be mixed among comments? And it helps other users to grasp information quicker imho

Reply:
But it's another thing that you have to do/to remember
And it's not so important if you have good commits history

Reply:
It’s still easier than refining commit history (which still should be done) & I believe we can decide if we need this by voting in our chat